### DISCUSSION TITLE
Preliminary program for version 3.1

### DISCUSSION THREAD

**Discussion by Serrof** [Hipparchus-Math/hipparchus#324] [General]
It is my opinion that we should ship the next minor release ASAP. The [milestone](https://github.com/Hipparchus-Math/hipparchus/milestone/4) is complete pending some last effort to cleanly integrate the nonlinear programming contribution. Anything you guys want to add?

**Comment by maisonobe** [comment]
Perhaps fixing a few easy ones like #269, #277 and #289?

**Reply by Serrof** [reply]
Sounds fair. I can take on #269

**Comment by Serrof** [comment]
#328 also since @MaximeJo is providing a non API breaking solution?

**Reply by maisonobe** [reply]
Agreed

**Comment by maisonobe** [comment]
I also want to add another feature (working on it this afternoon): Gauss-Markov process generation.

**Comment by maisonobe** [comment]
I think we are ready for release now.
Does everybody agree?
Does someone volunteer to perform the release?

**Reply by Serrof** [reply]
Are we considering #296 as done?

**Comment by maisonobe** [comment]
I would say yes.
It is still experimental code, though, perhaps we could add a warning in the javadoc, but it is really a great leap forward!
